vertx-mssql-client/src/main/java/io/vertx/mssqlclient/impl/codec/ExtendedQueryCommandCodec.java

```diff
@@ -207,6 +208,8 @@ private void encodeParamValue(ByteBuf payload, Object value) {
       encodeDateNParameter(payload, (LocalDate) value);
     } else if (value instanceof LocalTime) {
       encodeTimeNParameter(payload, (LocalTime) value, (byte) 6);
+    } else if (value instanceof LocalDateTime) {
+      encodeDateTimeNParameter(payload, (LocalDateTime) value, (byte) 6);
```

Comment:
Any opinions about the scale? I'm not sure if I ever got it right to hardcode it as 6 to not truncating any nano seconds in the `LocalTime` parameter.

Reply:
I filed #752 to make sure we analyze and take any action needed.